package.json

```diff
@@ -1,35 +1,22 @@
 {
-  "name": "data-union-solidity",
-  "description": "Smart contracts for Data Union with sidechain accounting",
+  "name": "data-union",
+  "description": "Smart contracts for Data Union with sidechain accounting and theGraph subgraph",
   "version": "0.0.1",
-  "scripts": {
-    "clean": "rm -rf build/ && mkdir -p build/contracts",
-    "build": "truffle compile && ./flatten",
-    "deploy": "truffle migrate",
-    "lint": "eslint . && solhint -f unix contracts/**/*.sol",
-    "format": "prettier --no-color --config .prettierrc --write contracts/**/*.sol",
-    "format-check": "prettier --no-color --config .prettierrc --check contracts/**/*.sol",
-    "coverage": "truffle run coverage --file=test/contracts/*",
-    "unit-test": "truffle test test/contracts/*",
-    "integration-test": "mocha test/e2e --exit",
-    "test": "npm run unit-test && npm run integration-test"
-  },
   "author": "Streamr Network AG",
   "license": "AGPL-3.0",
-  "devDependencies": {
-    "debug": "4.1.1",
-    "eslint": "7.22.0",
-    "openzeppelin-solidity": "3.1.0",
-    "solhint": "3.3.4",
-    "solidity-coverage": "^0.7.16",
-    "truffle": "^5.2.5",
-    "truffle-flattener": "^1.4.3",
-    "web3": "^1.2.4"
-  },
-  "dependencies": {
-    "@openzeppelin/contracts": "^3.4.1",
-    "ethers": "^5.0.32",
-    "@uniswap/v2-core": "^1.0.1",
-    "@uniswap/v2-periphery": "^1.1.0-beta.0"
+  "workspaces": [
+    "./packages/*"
+  ],
+  "scripts": {
+    "clean": "npm run clean -ws",
+    "build": "npm run build -ws",
+    "lint": "npm run lint -ws",
+    "format-check": "npm run format-check -w=data-union-solidity",
+    "test": "npm run test -w=data-union-solidity",
+    "integration-test": "npm run integration-test -w=data-union-solidity",
+    "graph1_build": "npm run deploy -w=data-union-solidity",
+    "graph2_copyInfo": "./scripts/copyAbis.sh",
+    "graph3_build": "npm run doAll -w=streamregistry-thegraph-subgraph",
+    "graph": "npm run graph1_build && npm run graph2_copyInfo && npm run graph3_build"
```

Comment:
Probably these steps aren't really necessary as separate?

Reply:
yes, they are copied from the network-contracts, just there for convenience or as a base for future scripting